clab/clab.go

```diff
@@ -501,6 +501,7 @@ func (c *CLab) CreateLinks(ctx context.Context, workers uint) {
 			}
 			c.m.Unlock()
 		}
+		time.Sleep(time.Duration(500) * time.Millisecond)
```

Comment:
```suggestion

		// prevent clab from throttle CPU when nodes take considerable time to start
		time.Sleep(500 * time.Millisecond)
```

Reply:
yepp looking fine... did not look up how to properly construct it... my bad!